.isort.cfg

```diff
@@ -19,5 +19,5 @@ multi_line_output=3
 order_by_type=True
 sections=FUTURE,STDLIB,THIRDPARTY,QT,LOCALFOLDER,RESOURCES,PICARD_UI
 skip_glob=**/picard/ui/ui_*.py
-skip=picard/const/attributes.py,picard/const/countries.py,picard/resources.py,scripts/picard.in
+skip=./tagger.py,picard/const/attributes.py,picard/const/countries.py,picard/resources.py,scripts/picard.in
```

Comment:
I tried that, but isort still insisted on moving the two imports `from picard.*` above my newly added code.

The actual best way to solve this would probably be to refactor the logging so that it does nothing on import and only gets initialized explicitly later on. The problem seems to be that at some point a default logger gets initialized and if we import picard.log before setting up stdout / stderr it will not take up the new streams.

Reply:
Ok, then keep it as is for now ;)